chunkey/encode_pipeline.py

```diff
@@ -144,39 +144,34 @@ def _scalar_commands(self, profile):
         """
         Pad videos with differing aspect ratios,
         either in pillar or letter box
-        """
+        """  # pylint: disable=pointless-string-statement
```

Comment:
Maybe it would be cleaner to just use the `#` format for the comments above, and remove this pylint directive?